blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/credential/CredentialApi.java

```diff
@@ -32,6 +33,8 @@
 
     private  final CredentialsStoreAction credentialStoreAction;
     private  final Reachable parent;
+    public static final String DOMAIN_NAME = "blueocean-domain";
```

Comment:
@stephenc thats right, the way its is currently, GitHub credentials are stored in 'github-domain' got for git keys, UI/client can create credentials in domain of their choice. 'blueocean-domain' is default domain where they get saved. Is this problem?